src/pdsfile/holdings_maintenance/pds3/pdschecksums.py

```
@@ -301,7 +302,7 @@ def write_checksums(check_path, abspairs, *, logger=None, limits=None):
             if '/.' in abspath:                     # flag invisible files
                 logger.invisible('Invisible file', abspath)
 
-            f.write('%s  %s\n' % (hex, abspath[lskip:]))
+            f.write('%s  %s\n' % (md5, abspath[lskip:]))
```

Ruff (0.16.1)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)